src/components/Header/SearchBar.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from "react";
+import "./header.css";
+
+const SearchBar = ({handleSearch}) => {
+  const [searchInput, setSearchInput] = useState("");
+  useEffect(() => {
+    if (searchInput===""){
+      // handleSearch("")
+    }
+  }, [searchInput])
+  return (
+    <form className="search-bar-container">
+      <input
+        type="text"
+        placeholder=""
+        value={searchInput}
+        onChange={(e) => setSearchInput(e.target.value)}
+      />
+      <i className="fa fa-search" style={{cursor:'pointer'}} onClick={()=>handleSearch(searchInput)}> </i>
```

Comment:
suggestion : onClick can be placed in a <span> tag, inside which you can keep the icon

Reply:
Yes, that could be done . But the only intention here was to keep the DOM elements as low as possible.